helm/cert-manager-app/charts/cert-manager-app/templates/cainjector-rbac.yaml

```diff
@@ -48,7 +48,7 @@ subjects:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
-  name: {{ template "certManager.name.cainjector" . }}:leaderelection
+  name: {{ template "certManager.name.cainjector" . }}--leaderelection
```

Comment:
I feel that renaming resources has the potential to cause problems when upgrading the app, but I suspect that this will just leave the old ones lying around and create the new resources afresh. Not certain without testing an upgrade though.

Reply:
I changed this because linter complained. But this seemes to apply just fine on clusters. You're probably right that this will leave around the old resources. 

I'll see if its possible to shush the linter for these